.evergreen/config.yml

```diff
@@ -1694,6 +1721,22 @@ tasks:
         - func: "run aws auth test with aws EC2 credentials"
         - func: "run aws ECS auth test"
 
+    - name: "aws-auth-test-6.0"
```

Comment:
Still need to add these two new tasks to the "aws-auth-test*" matrices.

Reply:
Ah ok, done.